### ISSUE TITLE
Color contrast issue in FeedValidator should be a warning, not an error.

### ISSUE THREAD

**Issue by bdferris** [google/transitfeed#46]
_From [joe.hugh...@gmail.com](https://code.google.com/u/105329591737206182947/) on March 19, 2008 16:51:41_

Also, the text of the issue description should be properly spaced so that
it doesn't run together when displayed in HTML.

_Original issue: http://code.google.com/p/googletransitdatafeed/issues/detail?id=46_
